go/pkg/client/cas_download.go

```diff
@@ -129,6 +140,11 @@ func (c *Client) DownloadOutputs(ctx context.Context, outs map[string]*TreeOutpu
 		if err := cache.Update(absPath, md); err != nil {
 			return fullStats, err
 		}
+		if c.diskCache != nil {
+			if err := c.diskCache.StoreCas(output.Digest, absPath); err != nil {
```

Comment:
Do you think it could be done as part of DownloadFiles() ? Otherwise this adds a serial latency.